Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.7-slim-buster
+
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update \
+  # dependencies for building Python packages
+  && apt-get install -y build-essential libgdal-dev python3-dev \
+  # psycopg2 dependencies
+  && apt-get install -y libpq-dev \
+  # Translations dependencies
+  && apt-get install -y gettext \
+  # cleaning up unused files
+  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN addgroup --system django \
+    && adduser --system --ingroup django django
+
+# Requirements are installed here to ensure they will be cached.
+COPY ./requirements /requirements
+RUN pip install --no-cache-dir -r /requirements/production.txt \
+    && rm -rf /requirements
+
+COPY ./start /start
+RUN sed -i 's/\r$//g' /start
```

Comment:
@MansoorMajeed what does this do?

Reply:
@anroopak To remove carriage returns if any